ludwig/datasets/mixins/process.py

```diff
@@ -46,6 +46,9 @@ def read_file(self, filetype, filename, header=0):
         elif filetype == 'tsv':
             file_df = pd.read_table(
                 os.path.join(self.raw_dataset_path, filename))
+        elif filetype == 'data':
```

Comment:
not sure, what's the difference between this and csv?

Reply:
Oh yes, I'll combine this with csv